### PR TITLE
Update Mypy settings descriptions

### PR DESCRIPTION
**Pull request by luabud**

### CODE REVIEW

File: package.nls.json [hunk 1]
```diff
@@ -17,4 +19,4 @@
     "settings.showNotifications.onError.description": "Show notifications for errors.",
     "settings.showNotifications.onWarning.description": "Show notifications for errors and warnings.",
     "settings.showNotifications.always.description": "Show notifications for errors and warnings."
-}
+}
```
Comment (karthiknadig) [inline package.nls.json]:
```suggestion
}

```

File: package.json [hunk 2]
```diff
@@ -233,4 +233,4 @@
         "webpack": "^5.89.0",
         "webpack-cli": "^5.1.4"
     }
-}
+}
```
Comment (karthiknadig) [inline package.json]:
```suggestion
}

```

**Review by karthiknadig — APPROVED** [review]

**Review by cwebster-99 — APPROVED** [review]
LGTM

**Review by karthiknadig — APPROVED** [review]

**Comment by karthiknadig** [comment]
@luabud you are making changes after merge. It wont get applied.

**Pull request merged**